tracer/src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -59,6 +61,8 @@ internal sealed class AspNetCoreDiagnosticObserver : DiagnosticObserver
         private readonly Tracer _tracer;
         private readonly Security _security;
         private readonly Iast.Iast _iast;
+        private readonly LiveDebugger _liveDebugger;
+        private readonly SpanCodeOriginManager _spanOrigin;
```

Comment:
nit: 
```suggestion
        private readonly SpanCodeOriginManager _spanOriginManager;
```

Reply:
Fixed in a5a2c085c57f23a246a2b781dec1bc9354502a16